src/openagi/data_extractors/data_loaders.py

```diff
@@ -0,0 +1,149 @@
+import json
+import tempfile
+from tempfile import NamedTemporaryFile
+from typing import Any
+from urllib.parse import urlparse
+
+import requests
+from bs4 import BeautifulSoup as Soup
+from langchain.docstore.document import Document
+from langchain_community.document_loaders import (
+    AirbyteStripeLoader,
+    GitLoader,
+    PyPDFLoader,
+    RecursiveUrlLoader,
+    TextLoader,
+    UnstructuredMarkdownLoader,
+    UnstructuredWordDocumentLoader,
+    WebBaseLoader,
+    YoutubeLoader,
+)
+from openagi.data_extractors.data_types import DataType
+from openagi.data_extractors.data_source import DataSource
+
+class DataLoader:
```

Comment:
create separate files for each type of files.
Create these loaders as actions, so that it can be used with actions or loaders.

Reply:
@shreehari-aiplanet Okay.

Reply:
Done